lib/action_controller/serialization.rb

```diff
@@ -27,6 +27,7 @@ def get_serializer(resource, options = {})
       end
       ActiveModel::SerializableResource.serialize(resource, options) do |serializable_resource|
         if serializable_resource.serializer?
+          serializable_resource.url_helper = self
```

Comment:
That's right. Any way we can at least narrow the interface to not include the entire controller? This is just be being paranoid of scope creep enabled by it being easy to do.  I like it when tools make it hard to do things the authors don't want you to :)